BBHSE/table.sql

```diff
@@ -0,0 +1,89 @@
+CREATE TABLE IF NOT EXISTS form_bbhse (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `date` datetime DEFAULT NULL,
+  `pid` bigint(20) DEFAULT NULL,
+  `user` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+  `groupname` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+  `authorized` tinyint(4) DEFAULT NULL,
+  `activity` tinyint(4) DEFAULT NULL,
+  `itime_in` mediumtext COLLATE utf8_unicode_ci,
```

Comment:
No.  and unicode_ci is stupid now.  General is faster, and MB4 is the nifty newness.